Documentation/contributing.rst

```diff
@@ -565,6 +582,27 @@ Best Practices for Writing Tests
         vm.ReportFailed()
     }
 
+
+Ginkgo Extensions
+~~~~~~~~~~~~~~~~~~
+
+In cilium, some Ginkgo features are extended to cover some uses cases that are
+useful for how cilium run test
+
+BeforeAll
+^^^^^^^^^^
+
+This function will run before all BeforeEach within a describe or context. This
+function is the new ``SetUp`` or initalize function for our Ginkgo test.
+
+AfterAll
+^^^^^^^^^
+
+This function will run after all ``AfterEeach`` functions defined in a Describe or
+Context. This function is the ``tearDown`` or the ``finilize`` function. The best
```

Comment:
Change to: A good use case for using `AfterAll` is to remove containers or pods that are needed for multiple `It`s.